action.yml

```
@@ -53,8 +53,13 @@ inputs:
   fix:
     description: 'Run with --fix? Defaults to false'
     required: false
+  npmInstall:
+    description: 'Run npm install (or yarn) for you. If you do not use this option, do the npm install step yourself, before running this action. Defaults to true'
+    required: false
+    default: true
```

With these changes then we can default this to false since it would now be only running if it does not detect a `node_modules` directory.

The only question would be if there are situations that this would run where `node_modules` would not exist at the top level... but that may be such a small set of cases that it isn't worth losing the automation over?

```suggestion
    default: false
```

---

I guess the only way to really find out is to wait until people start using the action more and more 🙂